code/game/gamemodes/vampire/vampire_powers.dm

```diff
@@ -298,17 +301,17 @@
 	M.current.visible_message("<span class='warning'>[M.current.name] bites [C.name]'s neck!</span>", "<span class='warning'>You bite [C.name]'s neck and begin the flow of power.</span>")
 	C << "<span class='sinister'>You feel the tendrils of evil [VAMP_CHARISMA in M.vampire.powers ? "aggressively" : "slowly"] invade your mind.</span>"
 	if(!ishuman(C))
-		M.current << "<span class='warning'>You can only enthrall humans.</span>"
+		M.current << "<span class='warning'>You can only enthrall humanoids.</span>"
 		return
 
 	if(M.current.can_enthrall(C) && do_mob(M.current, C, (VAMP_CHARISMA in M.vampire.powers) ? 25 : 50)) //takes half the time with Charisma unlocked
 		if(!M.current.can_enthrall(C))
 			M.current << "<span class='warning'>Either you or your target moved, and you couldn't finish enthralling them!</span>"
 			return
-		if(!M.current.vampire_power(300, 0)) // recheck
+		if(M.current.vampire_power(300, 0)) // recheck
```

Comment:
Here was the problem, for those curious.

Before it said:
If you don't have the blood necessary, then make the thrall. (doesn't work even without the blood due to var/mob/living/carbon/C = M.current.vampire_active(300, 0, 1) )

Now it says:
If you do have the blood neccessary, then make the thrall.